meinberlin/apps/plans/assets/PlansList.jsx

```diff
@@ -130,9 +130,13 @@ class PlansList extends React.Component {
   }
 
   getDate (item) {
-    let newDate = Date.parse(item.future_phase.replace(/ /g, 'T'))
-    newDate = Moment(newDate).format('DD.MM.YYYY')
-    return newDate
+    const date = Date.parse(item.future_phase.replace(/ /g, 'T'))
+    const dateStyle = {
+      year: 'numeric',
+      month: '2-digit',
+      day: '2-digit'
+    }
+    return toLocaleDate(date, undefined, dateStyle)
```

Comment:
Undefined just means that it takes the default of toLocaleDate?

Reply:
we could use the browser language but then mb is only in german anyway, so I guess we better stick with that?